modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm

```diff
@@ -54,6 +54,7 @@
 	var/bloodsucker_level = 0
 	var/bloodsucker_level_unspent = 1
 	var/additional_regen
+	var/blood_over_cap
```

Comment:
start this out as 0 or some default number

Reply:
Done so